src/sentry/static/sentry/app/components/eventOrGroupHeader.jsx

```diff
@@ -44,13 +44,14 @@ class EventOrGroupHeader extends React.Component {
     const basePath = `/organizations/${orgId}/issues/`;
 
     if (includeLink) {
-      const locationWithProject = {...this.props.location};
-      const query =
-        locationWithProject.query.project !== undefined
-          ? {
-              query: this.props.query,
-            }
-          : {query: this.props.query, _allp: 1}; //This appends _allp to the URL parameters if they have no project selected ("all" projects included in results). This is so that when we enter the issue details page and lock them to a project, we can properly take them back to the issue list page with no project selected (and not the locked project selected)
+      const locationWithQueries = {...this.props.location};
```

Comment:
Why do you need a clone of the location? You don't mutate the copy.

Reply:
This is old code, but yeah does not look like we need to clone this.